package.json

```diff
@@ -40,9 +48,8 @@
   },
   "husky": {
     "hooks": {
-      "commit-msg": "secret-squirrel-commitmsg",
-      "pre-commit": "secret-squirrel",
-      "pre-push": "make verify -j3"
+      "commit-msg": "dotcom-tool-kit git:commitmsg",
+      "pre-commit": "dotcom-tool-kit git:precommit && secret-squirrel"
```

Comment:
**chore:** The `@dotcom-tool-kit/secret-squirrel` plugin will configure the `git:precommit` task to run `SecretSquirrel`. Therefore we should not need to add `&& secret-squirrel` to the `pre-commit` hook.